test/correctness/specialize.cpp

```diff
@@ -232,7 +232,7 @@ int main(int argc, char **argv) {
         f.set_custom_trace(&my_trace);
 
         // Check bounds inference is still cool with widths < 8
-        f.infer_input_bounds(5);
+        f.infer_input_bounds(std::vector<int>{5});
```

Comment:
yuck. What goes wrong with {5}? Is it just the wrong type (which we could adapt for), or some deeper error?

Reply:
Oh, you explain above. Hrm.

Reply:
Looks like an overload that takes a single int would fix it, if we're willing to do that. https://stackoverflow.com/questions/46565165/single-element-vector-initialization-in-a-function-call

Reply:
Yeah, that would 'fix' it, but would also defeat the appeal of having a single well-defined calling convention for this. (How this wart ended up in place and apparently remains in place in C++17/C++20 is mind-boggling to me. Principle Of Most WTF apparently applies here.)